R/cfb_betting_lines.R

```diff
@@ -157,10 +158,10 @@ cfb_betting_lines <- function(game_id = NULL,
           dplyr::rename(game_id = .data$id) %>% 
           as.data.frame()
       }
-      message(glue::glue("{Sys.time()}: Scraping betting lines data..."))
+      # message(glue::glue("{Sys.time()}: Scraping betting lines data..."))
```

Comment:
Are we ever going to bring this back?

Reply:
If not, I'd just say delete it.

Reply:
On this one, probably not. I leave the error in there for now, but I was trying to think of any clever ways to message this that aren't overly verbose. I can delete